lmdeploy/cli/cli.py

```diff
@@ -49,6 +49,29 @@ def convert(self,
                 quant_path=quant_path,
                 group_size=group_size)
 
+    def list(self, is_torch: bool = False):
+        """List supported model names.
+
+        Examples 1:
+            lmdeploy list
+
+        Examples 2:
+            lmdeploy list --is_torch True
+
+        Args:
+            is_torch (bool): A flag to list model names for
```

Comment:
lmdeploy list --engine turbomind 
lmdeploy list --engine pytorch

turbomind is the default

Reply:
@lvhan028 
Ok. Do we consider list the prompt template for each model in the future? If so, we could make `lmdeploy list` as a command group and have two sub commands:
- `lmdeploy list model --engine pytorch`
- `lmdeploy list prompt_template model_name input_str ...`

Reply:
I haven't figured it out yet. Currently, let's use `lmdeploy list` to list all the supported models. 
For the detailed chat template, we can discuss it further

Reply:
OK. alright.